src/routes/home/Home.js

```diff
@@ -1050,7 +1050,7 @@ class Home extends React.Component {
                   });
               }}
             >
-              <fieldset disabled={this.state.isSubmitting}>
+              <fieldset disabled={this.state.isSubmitting && this.state.formatted_address !== 'Finding Address...'}>
```

Comment:
This only disables the Submit button if the address hasn't yet been found, but the desired behavior is:

> so yeah i think if it cannot resolve an address, then it should block the submission and give a clear explanation like “The location is not resolving to an address that the 311 form will allow. It requires a building number and street name and this sometimes doesn’t work in places like highways, airports or parks.”

So ideally we would know once the `process_validation` call has failed (need to check error handling behavior here) and show a toast message. A stopgap instead of this might be to instead not-disable the button when the `formatted_address` says "Finding Address...", but have the submit handler detect this situation and show the toast, rather than going forward with the submission.

Reply:
I'm not convinced we need this at all, and that the Finding Address submissions may be the result of occasional rate-limiting/blocking applied to the Heroku server, see here for details: https://reportedcab.slack.com/archives/C9VNM3DL4/p1677969946066369?thread_ts=1676288095.715499&cid=C9VNM3DL4